app/assets/javascripts/ace.js.erb

```diff
@@ -0,0 +1,11 @@
+//= require ace/ace
+
+<% modes = ENV['ACE_EDITOR_MODES'].presence || 'javascript,coffee,markdown,sql,json,yaml,text' %>
```

Comment:
@dsander , can we save ability to configurable?
against It works on self-build docker images only..
I'd like to keep huginn up to date and manage to ace without modifying core.